### PR TITLE
229 fix broken headings

### LINKED ISSUE #229: Headings broken in blog
seen in 
- http://hood.ie/blog/meet-the-hoodies-jenn-turner.html
- http://hood.ie/blog/Charlotte-Spencer-meet-the-hoodies-2.html
- http://hood.ie/blog/Alex-Feyerke-meet-the-hoodies-1.html

Odd, I’m quite sure this worked before?

### PR DESCRIPTION
**Pull request by NickColley**
closes #229

I've grepped these out so I think I've found them all.

### CODE REVIEW

File: _posts/2015-04-17-triangular-performance-tgif-66.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ npm now has [Private Modules](https://www.npmjs.com/private-modules)
 [Developing a tabs component in Angular 2](http://blog.thoughtram.io/angular/2015/04/09/developing-a-tabs-component-in-angular-2.html)
 
 
-## Events
+##  Events
```
Comment (varjmes) [inline _posts/2015-04-17-triangular-performance-tgif-66.md]: unnecessary extra space here

File: _posts/2015-04-30-art-of-deploying-bonbons-tgif-67.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ Options for [deploying ECMAScript 6](http://www.2ality.com/2015/04/deploying-es6
 
 > Adding a caching layer can indeed improve application performance, but it has its price and you need to be aware of it.<cite>[Things to consider before jumping to enterprise caching](http://vladmihalcea.com/2015/04/16/things-to-consider-before-jumping-to-enterprise-caching)</cite>
 
-## Events
+##  Events
```
Comment (varjmes) [inline _posts/2015-04-30-art-of-deploying-bonbons-tgif-67.md]: unnecessary extra space here

File: _posts/2015-05-29-unbossing-mario-tgif-68.md [hunk 3]
```diff
@@ -60,7 +60,7 @@ There’s been a larger discussion going on about the future of the web. The bes
 
 [The best icon is a text label](http://thomasbyttebier.be/blog/the-best-icon-is-a-text-label).
 
-## Events
+##  Events
```
Comment (varjmes) [inline _posts/2015-05-29-unbossing-mario-tgif-68.md]: unnecessary extra space here

File: _posts/2015-06-18-introverted-balloon-animals-tgif-69.md [hunk 4]
```diff
@@ -75,7 +75,7 @@ Video: [Engineering for the long game](https://www.youtube.com/watch?v=p0jGmgIrf
 
 
 
-## Events
+##  Events
```
Comment (varjmes) [inline _posts/2015-06-18-introverted-balloon-animals-tgif-69.md]: unnecessary extra space here

File: _posts/2015-06-18-introverted-balloon-animals-tgif-69.md [hunk 5]
```diff
@@ -87,7 +87,7 @@ Video: [Engineering for the long game](https://www.youtube.com/watch?v=p0jGmgIrf
 
 [CSSconf EU](http://http://2015.cssconf.eu), September 26th, Berlin, Germany *TICKETS ON SALE NOW, CALL FOR SPEAKERS OPEN*
 
-### Events from our friends:
+## # Events from our friends:
```
Comment (varjmes) [inline _posts/2015-06-18-introverted-balloon-animals-tgif-69.md]: this heading is broken as the space is in the wrong place :)

**Comment by varjmes** [comment]
@nickcolley wow, this is an epic PR. I went through every line and only notice 4 things to be fixed. great work! Once those bits are fixed this LGTM :+1:

**Comment by NickColley** [comment]
Cheers for reviewing, I've fixed those up.

me looking at that diff

File: _posts/2015-04-10-crucial-pong-tgif-65.md [hunk 6]
```diff
@@ -48,7 +48,7 @@ A few useful things to know about [machine learning](http://homes.cs.washington.
 
 Adding [authentication](https://auth0.com/blog/2015/04/09/adding-authentication-to-your-react-flux-app/) to React Flux apps
 
-## Events
+##  Events
```
Comment (varjmes) [inline _posts/2015-04-10-crucial-pong-tgif-65.md]: I missed one, sorry :sob:

Comment (NickColley) [inline _posts/2015-04-10-crucial-pong-tgif-65.md]: :P we'll get there.

**Comment by gr2m** [comment]
nice, thanks @nickcolley, LGMT :+1: 
I think the commit messages are good, too. We don’t need to follow the conventions as we do in the code repos

**Comment by varjmes** [comment]
merged. let's see if this works :)

**Comment by gr2m** [comment]
looks good now, thanks :)
- http://hood.ie/blog/meet-the-hoodies-jenn-turner.html
- http://hood.ie/blog/Charlotte-Spencer-meet-the-hoodies-2.html
- http://hood.ie/blog/Alex-Feyerke-meet-the-hoodies-1.html

**Pull request merged**